src/main/resources/static/templates/SSRFVulnerability/LEVEL_1/SSRF.html

```diff
@@ -1,15 +1,15 @@
 <div id="ssrfLevel1">
-    <h1 id="title">Welcome to your personal online image viewer!</h1>
-    <p id="message">Here you can give us a link and we will put your image down below!</p>
+    <h1 id="title">Welcome to your personal online file viewer!</h1>
+    <p id="message">Here you can give us a link and we will display your file down below!</p>
 
 
     <div id="input">
-        <input type="text" id="urlInput" placeholder="Image URL"/>
-        <button id="showBtn">Show Image</button>
+        <input type="text" id="urlInput" placeholder="File URL"/>
+        <button id="showBtn">Load File</button>
     </div>
 
     <div>
-        <div id="imageResponse" ></div>
+        <div id="fileResponse" ></div>
     </div>
 </div>
```

Comment:
we will just have a simple html page showing list of projects by SasanLabs (let's create a github gist for that) and that html page for each of the levels will send a call to backend and backend will call that gist page.
The 2nd last level will validate the url and last one will not take url as input. (both of the are secure)